doc/interoperability/pandas.rst

```diff
@@ -0,0 +1,295 @@
+  ...                         penalty='l2', C=1.0,
+  ...                         solver='lbfgs', multi_class='auto')
+  >>> lg.fit(train[feature_columns_train], train['target'])
+  LogisticRegression(C=1.0, class_weight=None, dual=False, fit_intercept=True,
+                     intercept_scaling=1, l1_ratio=None, max_iter=100,
+                     multi_class='auto', n_jobs=4, penalty='l2', random_state=123,
+                     solver='lbfgs', tol=0.0001, verbose=0, warm_start=False)
+  >>>
+  >>> res1 = lg.predict(test[feature_columns_train])
+  >>> res1[:5]
+  array([1, 0, 2, 1, 1])
+  >>> # result is actually
+  >>> res2 = lg.predict(test[feature_columns_test])
+  >>> res2[:5]
+  array([0, 0, 2, 0, 0])
+
+
+At the time of writing, it is the users responsibility to ensure that the
+column ordering in the data used for training the estimator is the same as the
+ordering of the data used for prediction. There is an ongoing discussion
+whether or not this will change in the future and this
+`issue <https://github.com/scikit-learn/scikit-learn/issues/7242>`__ should be
+watched and used to update this paragraph in the future. A simple and straight-
+forward way of ensuring that column ordering and column labels are the same is
+using something like `df[list of column names]` to enforce the
+correct ordering.
+
+Handling Categorical data
+=========================
+
+For a general guide on how to get started with categorical features please refer
+to :term:`categocrical feature` and :ref:`preprocessing_categorical_features`.
+It is worth noting that as of :ref:`changes_0_20_3`, both
+:class:`~sklearn.preprocessing.OneHotEncoder` and
+:class:`~sklearn.preprocessing.OrdinalEncoder`
+support string or Categorical columns coming straight from pandas DataFrames.
+
+
+Dealing with heterogenous data
+==============================
+
+Many modern datasets used with Scikit-learn contain heterogenous data. For the
+purpose of adding bespoke preprocessing steps for separate columns, Scikit-
+learn provides an experimental :class:`~sklearn.compose.ColumnTransformer` API
+(:ref:`column_transformer`).
+This API (which might change in the future) allows the definition of different
+transformation steps to be applied to different columns in either arrays,
+sparse matrices or pandas DataFrames.
+
+Dealing with missing values
+===========================
+
+As per the glossary, most Scikit-learn primitives do not work with missing
+values. If they do, NaN is the preferred representation of missing values. For
+more details, see :term:`missing values`. Non-numeric data is now also supported
+via the ``'most_frequent'`` or ``'constant'`` of the
+:class:`~sklearn.impute.SimpleImputer` class. For details see :ref:`impute`.
+
+
+Sparse DataFrames Handling
+=============================
+
+.. note::
+  **Issue:**
+  ``Sparse DataFrames`` are not automatically converted to ``scipy.sparse``
+  matrices.
```

Comment:
Sparse DataFrames are also deprecated in Pandas, if I'm not mistaken.

Reply:
I was not able to find evidence, I thought they still exist? https://pandas.pydata.org/pandas-docs/stable/user_guide/sparse.html?highlight=sparsedataframe ?